### ISSUE TITLE
Error Searching for a Map Service

### ISSUE THREAD

**Issue by davidafuller01** [nextgis/quickmapservices#219]
Getting this when trying to search. At this point, the plugin in non-functioning.

```
2021-03-10T19:29:45     WARNING    warning:/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.8/site-packages/urllib3/connectionpool.py:842: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
              warnings.warn((
             
             traceback: File "/Users/uname/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/quick_map_services/qms_service_toolbox.py", line 474, in run
              results = self.searcher.get_geoservices(
              File "/Users/uname/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/quick_map_services/qms_external_api_python/api/api_v1.py", line 30, in get_geoservices
              return self._get_json(self.full_url(sub_url), params)
              File "/Users/uname/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/quick_map_services/qms_external_api_python/api/api_abstract.py", line 53, in _get_json
              response = get(url, params=_params, verify=False)
              File "/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.8/site-packages/requests/api.py", line 76, in get
              return request('get', url, params=params, **kwargs)
              File "/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.8/site-packages/requests/api.py", line 61, in request
              return session.request(method=method, url=url, **kwargs)
              File "/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.8/site-packages/requests/sessions.py", line 530, in request
              resp = self.send(prep, **send_kwargs)
              File "/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.8/site-packages/requests/sessions.py", line 643, in send
              r = adapter.send(request, **kwargs)
              File "/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.8/site-packages/requests/adapters.py", line 439, in send
              resp = conn.urlopen(
              File "/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.8/site-packages/urllib3/connectionpool.py", line 597, in urlopen
              httplib_response = self._make_request(conn, method, url,
              File "/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.8/site-packages/urllib3/connectionpool.py", line 343, in _make_request
              self._validate_conn(conn)
              File "/Applications/QGIS-LTR.app/Contents/MacOS/lib/python3.8/site-packages/urllib3/connectionpool.py", line 842, in _validate_conn
              warnings.warn((
```

QGIS version | 3.16.4-Hannover | QGIS code revision | cdb515a206
-- | -- | -- | --
Compiled against Qt | 5.14.2 | Running against Qt | 5.14.2
Compiled against GDAL/OGR | 3.2.1 | Running against GDAL/OGR | 3.2.1
Compiled against GEOS | 3.9.1-CAPI-1.14.2 | Running against GEOS | 3.9.1-CAPI-1.14.2
Compiled against SQLite | 3.31.1 | Running against SQLite | 3.31.1
PostgreSQL Client Version | 12.3 | SpatiaLite Version | 4.3.0a
QWT Version | 6.1.4 | QScintilla2 Version | 2.11.4
Compiled against PROJ | 6.3.2 | Running against PROJ | Rel. 6.3.2, May 1st, 2020
OS Version | macOS 10.16
Active python plugins | OnlineRoutingMapper; QuickOSM; quick_map_services; nominatim; latlontools; disconnected-islands; autoSaver; LocatePoints; processing; db_manager; MetaSearch
-- | -- | -- | --
QMS plugin Installed version | 0.19.11.1
-- | -- | -- | --

**Comment by simgislab** [comment]
can't reproduce it here

![изображение](https://user-images.githubusercontent.com/172806/110757683-b9c17e80-825c-11eb-8327-c56bc58952ca.png)

**Comment by drnextgis** [comment]
I'm getting the same warning but search still works:

```
2021-03-11T09:56:14     WARNING    warning:/usr/lib/python3/dist-packages/urllib3/connectionpool.py:860: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
              InsecureRequestWarning)
             
             traceback: File "/home/denis/.local/share/QGIS/QGIS3/profiles/default/python/plugins/quick_map_services/qms_service_toolbox.py", line 477, in run
              cumulative_status=self.status_filter
              File "/home/denis/.local/share/QGIS/QGIS3/profiles/default/python/plugins/quick_map_services/qms_external_api_python/api/api_v1.py", line 30, in get_geoservices
              return self._get_json(self.full_url(sub_url), params)
              File "/home/denis/.local/share/QGIS/QGIS3/profiles/default/python/plugins/quick_map_services/qms_external_api_python/api/api_abstract.py", line 53, in _get_json
              response = get(url, params=_params, verify=False)
              File "/usr/lib/python3/dist-packages/requests/api.py", line 72, in get
              return request('get', url, params=params, **kwargs)
              File "/usr/lib/python3/dist-packages/requests/api.py", line 58, in request
              return session.request(method=method, url=url, **kwargs)
              File "/usr/lib/python3/dist-packages/requests/sessions.py", line 520, in request
              resp = self.send(prep, **send_kwargs)
              File "/usr/lib/python3/dist-packages/requests/sessions.py", line 630, in send
              r = adapter.send(request, **kwargs)
              File "/usr/lib/python3/dist-packages/requests/adapters.py", line 440, in send
              timeout=timeout
              File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 601, in urlopen
              chunked=chunked)
              File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 346, in _make_request
              self._validate_conn(conn)
              File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 860, in _validate_conn
              InsecureRequestWarning)
```

**Comment by 514ckw4r3** [comment]
@davidafuller01, InsecureRequestWarning fixed in 0.19.15. This warning should not affect the functionality of the plugin. Please create a new issue with a detailed description if you have any problems with the functionality of the plugin.
